ChakraCore Samples/Hello World/Linux_OSX/Makefile

```diff
@@ -9,13 +9,16 @@ CC=g++
 LIBRARY_PATH=../BuildLinux/$(BUILD_TYPE)/lib
 LDIR=$(LIBRARY_PATH)/../pal/src/libChakra.Pal.a \
 	$(LIBRARY_PATH)/Common/Core/libChakra.Common.Core.a \
-	$(LIBRARY_PATH)/Jsrt/libChakra.Jsrt.a
+	$(LIBRARY_PATH)/Jsrt/libChakra.Jsrt.a \
```

Comment:
How about keeping this part as is and adding `libicu` libs under the `darwin` related part?

LDIR=$(LDIR) [icu libs here] ?